django/core/management/commands/test.py

```diff
@@ -79,8 +85,11 @@ def handle(self, *test_labels, **options):
             os.environ['DJANGO_LIVE_TEST_SERVER_ADDRESS'] = options['liveserver']
             del options['liveserver']
 
+
+        tags = set(options['tags']) if options['tags'] is not None else set()
+        exclude_tags = set(options['exclude_tags']) if options['exclude_tags'] is not None else set()
```

Comment:
Unfortunately it seems that we need to defend against `None`- after I changed this just to `.get('tags', set())` tests are still failing.